constructor/main.py

```diff
@@ -172,17 +178,13 @@ def main_build(dir_path, output_dir='.', platform=cc_platform,
             else:
                 env_config[config_key] = value
 
-    try:
-        exe_name, exe_version = identify_conda_exe(info.get("_conda_exe"))
-    except OSError as exc:
+    if exe_type is None or exe_version is None:
         logger.warning(
             "Could not identify conda-standalone / micromamba version (%s). "
-            "Will assume it is compatible with shortcuts.",
-            exc,
-         )
-        exe_name, exe_version = None, None
-    if sys.platform != "win32" and exe_name is not None and (
-        exe_name == "micromamba" or Version(exe_version) < Version("23.11.0")
+            "Will assume it is compatible with shortcuts."
```

Comment:
You are missing the exception here, so the `%s` placeholder above won't be filled in.